frontend/ui/data/onetime_migration.lua

```diff
@@ -218,9 +218,9 @@ if last_migration_date < 20210508 then
     DocCache:clearDiskCache()
 end
 
--- 20210518, ReaderFooter, https://github.com/koreader/koreader/pull/7702
```

Comment:
I'd keep the history ;)

(IIRC, there's another example of a function with two dates/PRs earlier).

Reply:
Whoops, I thought that commit was changing earlier thing from this PR.